modules/statics/src/coins.ts

```diff
@@ -1209,17 +1198,24 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset['avaxc:PNG']
   ),
   tavaxErc20(
-    'tavaxc:USDT.E',
+    'tavaxc:LINK',
+    'Test Chainlink',
+    18,
+    '0x0b9d5d9136855f6fec3c0993fee6e9ce8a297846',
+    UnderlyingAsset['avaxc:LINK']
+  ),
+  tavaxErc20(
+    'tavaxc:USDT',
     'Test Tether USD',
     6,
     '0xc7198437980c041c805a1edcba50c1ce5db95118',
```

Comment:
should this be updated to '0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7'?

Reply:
@shiksb could you confirm USDT and USDC testnet contract address?

Reply:
look ok